test/SmokeTest.php

```diff
@@ -72,18 +72,28 @@ public function requestProvider() : Traversable
         }
 
         yield '/annual-reports/2012 wrong version' => [
-            $this->createRequest('/annual-reports/2012', 'application/vnd.elife.annual-report+json; version=2'),
+            $this->createRequest('/annual-reports/2012', 'application/vnd.elife.annual-report+json; version=3'),
             'application/problem+json',
             406,
         ];
 
         yield $path = '/annual-reports' => [
             $this->createRequest($path),
+            'application/vnd.elife.annual-report-list+json; version=2',
+        ];
+        yield $path = '/annual-reports' => [
```

Comment:
We need unique keys here, generators can produce duplicates but not sure how this gets used (so one of these could be missed).

(Hence others having `version 1` etc suffixes.)

Reply:
good spot. I've amended the keys.